thetis/exporter.py

```diff
@@ -310,13 +310,18 @@ def export(self):
             sys.stdout.write('\n')
             sys.stdout.flush()
 
-    def export_bathymetry(self, bathymetry_2d):
+    def export_bathymetry(self, bathymetry_2d, solve_exner=False):
         """
         Special function to export 2D bathymetry data to disk
 
-        Bathymetry does not vary in time so this only needs to be called once.
+        If exner equation not used then bathymetry does not vary in time so this
```

Comment:
This seems inaccurate: all the argument `solve_exner` does is change the name of the bathymetry file. Is that really necessary?

Reply:
The reason I put this in was because before for any simulation, thetis exports the bathymetry at t = 0. I wanted to leave this in because this is useful functionality for users. However for people who are solving exner it is a bit misleading to just have one file called `bath.pvd`. It is not clear that this is the initial bathymetry, it could just as well be the final one or one of the ones in the middle. Therefore I renamed the file to make it clearer to the user.